CI-API.Data/Repository/AdminRepository.cs

```diff
@@ -855,18 +855,16 @@ public async Task<JsonResult> GetAllMissionApplication(string? search)
 
                 if (search == null)
```

Comment:
use string.isnullorempty
Also please change to all existing code as you notice

Reply:
changes done